src/wovnio/html/HtmlConverter.php

```diff
@@ -272,10 +272,10 @@ private function buildHrefLang($lang_code)
         if ($this->store->hasDefaultLangAlias()) {
             $url = $this->headers->removeLang($url, $this->store->defaultLang());
             $url = Url::addLangCode($url, $this->store, $lang_code, $this->headers);
-        } else {
-            if ($lang_code !== $this->store->defaultLang()) {
-                $url = Url::addLangCode($url, $this->store, $lang_code, $this->headers);
-            }
+        } elseif ($lang_code !== $this->store->defaultLang()) {
+            $url = Url::addLangCode($url, $this->store, $lang_code, $this->headers);
+        } elseif ($this->store->getCustomDomainLangs() !== null) {
```

Comment:
Since this no longer does `->getSource` stuff anymore, I think I have changed my mind on this and maybe we should combine the two ifs. However I don't really like using `$this->store->getCustomDomainLangs()` as a proxy for whether custom domain is enabled. It is possible for someone to have the settings in their config but still use path pattern etc. So I would prefer to just check the pattern name explicitly.

(the other usages of this for serializing the params etc is fine).

Reply:
Good catch.